constants.go

```diff
@@ -13,6 +13,13 @@ const (
 	TRACE   = "TRACE"
 )
 
+// Constants for PostForm
+const (
+	POSTFORM            = "POST"
+	POSTFORMHeaderKey   = "Content-Type"
+	POSTFORMHeaderValue = "application/x-www-form-urlencoded"
```

Comment:
The header values can be anything, We can't maintain those. I will suggest to create this constant on the function level.

Reply:
Added as input to r.WithHeaders() inside of the PostForm() function since this is [similar to the official http packages implementation](https://github.com/golang/go/blob/ad642727247383079c8546ca365172859641a800/src/net/http/client.go#L877)